specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/ManagedLedgerDigestUploads.json

```diff
@@ -0,0 +1,335 @@
+          },
+          "202": {
+            "description": "Disabling the ledger digest upload is in progress."
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
It is recommended to add "x-ms-long-running-operation-options" as "location" here and add headers to 202 response with "Location" in headers, to show that Post Async progress can be tracked with location header.

Reply:
This has not been addressed?